qa/pull-tester/rpc-tests.py

```diff
@@ -198,6 +192,17 @@ def main():
             # (for parallel running efficiency). This combined list will is no
             # longer sorted.
 
+    # Remove the test cases that the user has explicitly asked to exclude.
+    if args.exclude:
+        for exclude_test in args.exclude.split(','):
+            if exclude_test + ".py" in test_list:
```

Comment:
Don't you need the following?
```
            if exclude_test in test_list:
                test_list.remove(exclude_test)
            if exclude_test + ".py" in test_list:
                test_list.remove(exclude_test + ".py")
```

Otherwise --exclude won't work if you specify the test with including .py explicitly.
Oh, sorry, the --exclude documentation explains it, never mind.